.github/actions/get-issue-url-and-version-number-v1/README.md

```diff
@@ -0,0 +1,37 @@
+# get-issue-url-and-version-number-v1
+
+This GitHub Action gets the version number and issue url from the release candidate issue.
+
+## Outputs
+
+| Output           | Description                                                 |
+| ---------------- | ----------------------------------------------------------- |
+| `version-number` | The version number of the release candidate.                |
+| `issue-url`      | The url of the issue associated with the release candidate. |
+
+## Example usage
+
+```yaml
+name: Get version number and issue url and post to Slack
+
+on: push
```

Comment:
Should specify that we only want this to run when the release branch is pushed to

```suggestion
on: 
  push:
    branches:
      - release
```